<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(123, 175, 212);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Select your option:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>290</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Icon/OMCDb.ico</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="cultureOrder">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>570</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {

background-color: rgb(210, 210, 210);

}

QPushButton {

background-color: rgb(193, 193, 193);

}</string>
    </property>
    <property name="text">
     <string>Culture Order Forms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resultEntry">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>630</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {

background-color: rgb(210, 210, 210);

}

QPushButton {

background-color: rgb(193, 193, 193);

}</string>
    </property>
    <property name="text">
     <string>Result Entry</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rejectionLogForm">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>690</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {

background-color: rgb(210, 210, 210);

}

QPushButton {

background-color: rgb(193, 193, 193);

}</string>
    </property>
    <property name="text">
     <string>Rejection Log Form</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qaManager">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>750</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {

background-color: rgb(210, 210, 210);

}

QPushButton {

background-color: rgb(193, 193, 193);

}</string>
    </property>
    <property name="text">
     <string>QA Manager</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>1100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {

background-color: rgb(210, 210, 210);

}

QPushButton {

background-color: rgb(193, 193, 193);

}</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>1100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {

background-color: rgb(210, 210, 210);

}

QPushButton {

background-color: rgb(193, 193, 193);

}</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>cultureOrder</zorder>
   <zorder>label_4</zorder>
   <zorder>resultEntry</zorder>
   <zorder>rejectionLogForm</zorder>
   <zorder>qaManager</zorder>
   <zorder>logout</zorder>
   <zorder>settings</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
